### ISSUE TITLE
Loading Backbone LocalStorage with requirejs returns an error

### ISSUE THREAD

**Issue by scavassa91** [jeromegn/Backbone.localStorage#220]
I'm loading BackBone LocalStorage with requirejs and it returns this error

![image](https://user-images.githubusercontent.com/19716895/29075019-8ae4b5da-7c27-11e7-8f39-96ce8753fe1c.png)

I'm thinking that is related with the AMD verification in the beginning of the code, because the object is always undefined.

**Comment by scavassa91** [comment]
To fix this problem I've done this.

```
else if(typeof define === 'function' && define.amd)
	define(['underscore', 'backbone'], factory);
```

**Comment by scott-w** [comment]
Hi guys, I've closed this ticket and published v2.0.1 to NPM. Thanks for your patience and to the guys who published fixed outputs to (hopefully) keep people's builds working in the meantime 👍

**Comment by scavassa91** [comment]
Hello @scott-w, I've tested the solution but i noticed that underscore is named as backbone and this cause another problem.
![error](https://user-images.githubusercontent.com/19716895/29529784-6adb9394-8677-11e7-8b7a-a86c6dc130ea.jpg)
Would you like that I make a pull request to fix this issue?

**Comment by scott-w** [comment]
Hi @scavassa91 - thanks for this, I'll reopen the issue 👍

**Comment by scott-w** [comment]
@scavassa91 I've sorted this and published a new build to NPM @2.0.2. Thanks for the quick spot!
